Source/Core/Common/StringUtil.cpp

```diff
@@ -77,13 +77,10 @@ bool AsciiToHex(const std::string& _szValue, u32& result)
   // Set errno to a good state.
   errno = 0;
 
-  char* endptr = nullptr;
-  const u32 value = strtoul(_szValue.c_str(), &endptr, 16);
+  const u32 value = strtoul(_szValue.c_str(), nullptr, 16);
 
-  if (!endptr || *endptr)
-    return false;
-
-  if (errno == ERANGE)
+  // Fail if errno EINVAL or ERANGE
```

Comment:
According to the man strtoul(3):

ERANGE

> 
> The resulting value was out of range.
> The implementation may also set errno to EINVAL in case no conversion was performed (no digits seen, and 0 returned).